app/javascript/spec/dual-list-select/dual-list-select.spec.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import DualList from '../../components/dual-list-select';
+import List from '../../components/dual-list-select/list';
+
+const FieldProvider = ({ component, ...props }) => (
```

Comment:
I would move this to helper function for tests. I can image it being used for all custom form components!